researcher_ids/RI_Claims_V1.md

```diff
@@ -506,7 +506,9 @@ Although standard fields within a [Passport Visa Object](#passport-visa-object)
 are defined in this section, other fields MAY exist within the object
 and should be ignored by any Passport Clearinghouse that is not familiar
 with the use of such fields. Field names are reserved for definition by
-GA4GH (or a body it elects).
+GA4GH (or a body it elects). For field types that define their field values
+as a character strings, the string values MUST NOT exceed 255 characters in
```

Comment:
What is the purpose of such artificial limit? 255 characters is a short string for carrying data.

Reply:
Removed